specification/apimanagement/control-plane/Microsoft.ApiManagement/preview/2017-03-01/apimportalsettings.json

```diff
@@ -0,0 +1,472 @@
+            "$ref": "./apimanagement.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "Delegation settings was updated successfully."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./apimanagement.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "DelegationSettings"
+        ],
+        "operationId": "DelegationSettings_CreateOrUpdate",
+        "description": "Create or Update Delegation settings.",
+        "x-ms-examples": {
+          "ApiManagementPutPortalSettingsDelegation": {
+            "$ref": "./examples/ApiManagementPutPortalSettingsDelegation.json"
+          }
+        },
+        "parameters": [
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/PortalDelegationSettings"
+            },
+            "description": "Create or update parameters."
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "Delegation settings was successfully updated.",
+            "schema": {
+              "$ref": "#/definitions/PortalDelegationSettings"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./apimanagement.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "PortalSigninSettings": {
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "description": "Redirect Anonymous users to the Sign-In page."
+        }
+      },
+      "description": "Sign-In settings for the Developer Portal."
+    },
+    "PortalSignupSettings": {
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "description": "Allow users to sign up on a developer portal."
+        },
+        "termsOfService": {
+          "type": "object",
+          "$ref": "#/definitions/TermsOfServiceProperties",
+          "description": "Terms of service contract properties."
+        }
+      },
+      "description": "Sign-up settings contract properties."
+    },
+    "TermsOfServiceProperties": {
+      "properties": {
+        "text": {
+          "type": "string",
+          "description": "A terms of service text."
+        },
+        "enabled": {
+          "type": "boolean",
+          "description": "Display terms of service during a sign-up process."
+        },
+        "consentRequired": {
+          "type": "boolean",
+          "description": "Ask user for concent."
```

Comment:
>concent [](start = 39, length = 7)

consent

Reply:
will fix it.

---
In reply to: [163381532](https://github.com/Azure/azure-rest-api-specs/pull/2269#discussion_r163381532) [](ancestors = 163381532)

Reply:
fixed